packages/utilities/psammead-test-helpers/src/index.js

```diff
@@ -1,11 +1,11 @@
-import renderer from 'react-test-renderer';
+import { render } from 'react-testing-library';
 import 'jest-styled-components';
 import ShallowRenderer from 'react-test-renderer/shallow';
 
 export const shouldMatchSnapshot = (title, component) => {
   it(title, () => {
-    const tree = renderer.create(component).toJSON();
-    expect(tree).toMatchSnapshot();
+    const { asFragment } = render(component);
```

Comment:
Running this on psammead-headings I go from 
```
exports[`Headline component should render correctly 1`] = `
.c0 {
  color: #222222;
  font-family: ReithSerifNewsMedium,Helvetica,Arial,sans-serif;
  margin: 0;
  padding: 2rem 0 1rem 0;
  font-size: 1.75em;
  line-height: 2rem;
}

@media (min-width:20em) and (max-width:37.4375em) {
  .c0 {
    font-size: 2em;
    line-height: 2.25rem;
  }
}

@media (min-width:37.5em) {
  .c0 {
    font-size: 2.75em;
    line-height: 3rem;
  }
}

<h1
  className="c0"
>
  This is my headline.
</h1>
`;
```
to
```
exports[`Headline component should render correctly 1`] = `
<DocumentFragment>
  <h1
    class="sc-bdVaJa gMUPYM"
  >
    This is my headline.
  </h1>
</DocumentFragment>
```
It feels like were losing a lot of information here

Reply:
Interesting. 🤔 Will look into why it's not including the styles in this case, but was when I included it in Simorgh container. I'm not sure about the source of the inconsistency.